### PR TITLE
Complete contact section design

### PR DESCRIPTION
**Pull request by MAstrauskas**
At this stage, only the responsiveness of the contact form and design has to be completed. The form itself should not work for now.

### CODE REVIEW

File: src/assets/img/sections/contact/Minder.svg [hunk 1]
```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (pawelgrzybek) [inline src/assets/img/sections/contact/Minder.svg]: Optimise, optimise, optimise...

https://petercollingridge.appspot.com/svg-optimiser

File: src/assets/img/sections/contact/Minder.svg [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+<svg width="822px" height="567px" viewBox="0 0 822 567" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
+    <!-- Generator: sketchtool 55.1 (78136) - https://sketchapp.com -->
+    <title>ED1E1C88-CF3D-442A-9C2A-8D69AAA2F00D</title>
```
Comment (pawelgrzybek) [inline src/assets/img/sections/contact/Minder.svg]: Screen readers are going crazy here. Better to skip title tag than keeping something like dude.

File: src/assets/img/sections/contact/Minder.svg [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+    <desc>Created with sketchtool.</desc>
```
Comment (pawelgrzybek) [inline src/assets/img/sections/contact/Minder.svg]: Likewise.

File: src/js/index.js [hunk 4]
```diff
@@ -24,9 +24,35 @@ function toggleNav(e) {
 
 navButton.addEventListener("click", toggleNav);
 
+/* Closes mobile menu on pressed link */
+navLink.forEach(link =>
```
Comment (pawelgrzybek) [inline src/js/index.js]: The `forEach` on DOM collection is not incredibly well supported. The easiest fix here is to convert it to array, like so.

```js
[...navLink].forEach(callback);
```

https://pawelgrzybek.com/loop-through-a-collection-of-dom-elements/

😎

File: src/assets/img/sections/contact/Minder.svg (hunk 1 above)
Comment (MAstrauskas) [reply src/assets/img/sections/contact/Minder.svg]: Done

**Pull request merged**